client/src/components/admin/RoundDoubleCheck/RoundDoubleCheck.js

```diff
@@ -48,6 +48,13 @@ const ROUND_QUERY = gql`
         enteredAt
       }
     }
+    officialWorldRecords {
+      event {
+        id
+      }
+      type
+      attemptResult
+    }
```

Comment:
Just copied over this part of the query from `AdminRound.js`. Not sure if I should abstract this further into a fragment or not. Is that what a fragment is? I don't know GQL :P

Reply:
In this case I think it's fine without the fragment, it's brief and we use it only in two places.